### PR TITLE
Update tests for web 1.0

### PR DESCRIPTION
**Pull request by cgewecke**
Lowercasing some addresses in the tests because `web3 1.0` checksums them.

### CODE REVIEW

**Comment by gnidan** [comment]
Actually we can probably just merge this into `develop`, no?

**Comment by cgewecke** [comment]
@gnidan Ah actually I think this needs web3 bumped up, and the nonsense about lower-casing removed. . . haven't really looked at this for a while and the last build failure was just about syncing.

**Comment by cgewecke** [comment]
@gnidan Actually, just dawned on me that web3 is substantially more than a dev dep here. Do you think it should be bumped? Maybe it should be done in a separate step....

**Comment by gnidan** [comment]
@cgewecke separately from the rest of Truffle, you mean? or separate from this PR?

Either way should be fine - since the Debugger bundles web3 itself, we shouldn't end up with an extra web3 version in the main truffle build. Theoretically we could even upgrade to 1.0 on `develop` here.

**Comment by cgewecke** [comment]
@gnidan Yes agree - because it's quarantined in the build - just wondering if there might be issues upgrading and they'd be easier to detect if they weren't accompanied by changes in 10 other modules. You only pull addresses directly from the client through `web3` right? Only thing I could think of was that there might be checksum / non-checksum discrepancies if you were getting them from something else within Truffle like the artifact.

**Comment by gnidan** [comment]
> You only pull addresses directly from the client through web3 right?

Addresses come from two places: the tx receipt `to` field, and the stack references to any other contracts via `CALL`/`DELEGATECALL`. The second would not be checksummed, and the first would be whatever web3 provides in the receipt.

**Comment by cgewecke** [comment]
@gnidan Ah ok then I think web3 1.0 will require that the receipt addresses are converted to lower-case, but its an internal debugger concern.

File: test/context.js [hunk 1]
```diff
@@ -116,17 +116,17 @@ describe("Contexts", function () {
...
     assert.include(
-      affectedAddresses, outer.address,
+      affectedAddresses, outer.address.toLowerCase(),
       "OuterContract should be an affected address"
     );
 
     assert.include(
-      affectedAddresses, inner.address,
+      affectedAddresses, inner.address.toLowerCase(),
       "InnerContract should be an affected address"
```
Comment (cgewecke) [inline test/context.js:130]: Expected to be able to remove all the address normalization but looks like `affectedAddresses` are still not check-summed coming out. Not sure if that's a problem or not. CI failure for post web3 update: c58c93d1726f6efae46db6485c25548499e19cc7
```shell
AssertionError: InnerContract should be an affected address: expected [ Array(2) ] to include '0xF248a7B090E45FBAc267719a09aEeFeDB4b2cd40'
```

**Review by gnidan — APPROVED** [review]
Should we see if this makes `develop` blow up or not, also? Or just keep this on `next` so we can upgrade across the board

**Comment by cgewecke** [comment]
Maybe open this on branch targeting develop and check it out in the darqness first? Totally your call.

**Comment by gnidan** [comment]
Nah, let's not give this package special privileges :)

**Pull request merged**